source/api_reference/create_a_payment_reference/index.html.md.erb

```diff
@@ -0,0 +1,188 @@
+| `metadata` | object | Contains any custom metadata you sent when creating this payment. <br><br>The `metadata` object does not appear if you did not send any custom metadata in the request body.<br><br>You can [read more about adding custom metadata to payments](/custom_metadata). |
+| `email` | string | The paying user's email address. The paying user's email field will be prefilled with this value when they make their payment.<br><br>`email` does not appear if you did not include it in the request body. |
+| `state` | object | Contains information about where the payment is in its journey.<br><br>You can [read more about the payment status lifecycle](/api_reference/#payment-status-lifecycle). |
+| `state.status` | string | The status of this payment.<br><br>`status` value will automatically be set to `created` because you've just created this payment.<br><br>You can [read more about the meanings of each payment status](/api_reference/#payment-status-lifecycle). |
+| `state.finished` | boolean | Indicates whether a payment journey is finished. <br><br>`finished` will automatically be set to `false` because you've just created this payment.<br><br>A `finished` payment journey does not always mean the user has made a payment. For example, a user may submit payment details but their bank rejects the payment - the payment journey would be `finished` but the payment has not been made. |
+| `payment_id` | string | The unique ID GOV.UK Pay automatically associated with this payment when you created it. |
+| `payment_provider` | string | The payment service provider that will process this payment. |
+| `created_date` | date (ISO 8601) | The date and time you created this payment.<br><br>This value uses Coordinated Universal Time (UTC) and ISO 8601 format - `YYYY-MM-DDTHH:MM:SSZ`. |
+| `refund_summary` | object | Contains information about refunds on this payment, such as refund availability.<br><br>You can [read more about refunding payments](/refunding_payments). |
+| `refund_summary.status` | string | Whether you can refund the payment.<br><br>`status` value will be `pending` because your user has not yet made this payment.<br><br>You can [read about the meanings of each refund `status` value](/refunding_payments/#status). |
+| `refund_summary.amount_available` | number | How much you can refund to the user, in pence, once they have completed the payment. |
+| `refund_summary.amount_submitted` | number | How much you've already refunded to the user in pence. |
+| `settlement_summary` | object | Contains information about when this payment was settled.<br><br>Payments are settled when your payment service provider sends funds to your bank account. You can read further [guidance on when your payment service provider sends fund to your bank account](/reporting/#filter-payments-by-when-your-psp-sent-payments-to-your-bank-account).<br><br>`settlement_summary` will be empty because you've just created this payment. |
```

Comment:
```suggestion
| `settlement_summary` | object | Contains information about when this payment was settled.<br><br>Payments are settled when your payment service provider sends funds to your bank account. You can read further [guidance on when your payment service provider sends funds to your bank account](/reporting/#filter-payments-by-when-your-psp-sent-payments-to-your-bank-account).<br><br>`settlement_summary` will be empty because you've just created this payment. |
```